packages/core/demo/data/zoom-bar.ts

```diff
@@ -118,19 +118,47 @@ export const zoomBarLineTimeSeriesInitDomainOptions = addZoomBarToOptions(
 	Object.assign({}, timeSeriesAxisChart.lineTimeSeries15secondsOptions),
 	{ sliderView: true }
 );
-zoomBarLineTimeSeriesInitDomainOptions["title"] += " (initial zoomed domain)";
+zoomBarLineTimeSeriesInitDomainOptions.title += " (initial zoomed domain)";
 zoomBarLineTimeSeriesInitDomainOptions.zoomBar.top.initialZoomDomain = initialZoomDomain;
 
+export const zoomBarLockedData = [];
+export const zoomBarLockedOptions = addZoomBarToOptions(
+	Object.assign(
+		{
+			data: {
+				loading: true
+			},
+			// required for correct spacing after PR 785
+			legend: {
+				enabled: false
+			}
```

Comment:
why do we need to disable the legend here?

Reply:
In PR 785, the logic is changed to legend will not disable automatically when data is loading.
But when legend is enabled while data is loading, the space between zoom bar and skeleton is gone.
I don't know why the code logic is changed, so I have to disable legend here.
https://github.com/carbon-design-system/carbon-charts/pull/785

Reply:
It was changed because when disabling the legend in loading, if you disable loading then there will be no legend

Reply:
I see.
But if data is loading without disabling legend, the `Legend` and `legendSpacerComponent` components are still added into `fullFrameComponent `, and the `fullFrameComponentDirection` will be `LayoutDirection.COLUMN_REVERSE`.
In that case, the skeleton will be shifted up and the space between zoom bar and skeleton is gone.
That's why I disable legend here.

Reply:
well, is there a solution that works for both cases?

Reply:
Sure

Reply:
@theiliad After some investigation, I prefer to keep to disabled legend in demo data because:

1. This problem is related to #748 #785 #802 , and I think it's out of this PR scope.
2. #785 looks weird to me. Maybe I didn't get the original issue it likes to resolve, but why we want to create an empty legend and cause the possible space problem ?
3. Both `skeleton` and `legend` are in `full-frame` component in axis-chart.
`skeleton` width and height are set to parent component's size, which means `skeleton` didn't expect `legend` should be displayed while `skeleton` is displayed (data is loading).
If we want to show skeleton and legend at the same time, something needs to change.
If we want to disable legend while showing skeleton (data loading), we may need to roll back #785.
Again, I believe this is out of this PR scope.

Reply:
Legend option is removed from the demos in https://github.com/carbon-design-system/carbon-charts/pull/776/commits/765ae75541fdc1816c0688cef24a32e4acfc179b.